RULES.md

```diff
@@ -0,0 +1,94 @@
+# Traffic rules
+
+## TLDR
+We need to:
+
+Allow independent groups in the content project to work at their own pace on parts of the project that don’t influence each other.
+Introduce enter and exit criteria for roles with elevated privileges so the hierarchy can self-organize.
+Define a framework for organizations to join the project and assume responsibilities.
+Make sure that the project stays coherent and individual parts contribute to its value, and the project doesn’t become a disgusting spaghetti soup.
+
+
+## Decomposition
+
+In order to keep the project scalable, we divide the project into separate areas, and assign groups of users that can be granted commit access with the intention to develop content in respective areas.
+Those areas are going to be segmented by
+
+- Products (RHEL, Ubuntu, …) - make it easy to develop content of a product that doesn’t influence the rest of the build system.
+- Product-specific profiles (PSPs s.a. Firefox STIG, Ubuntu CIS, …) and respective control files - make it easy for SMEs that only want to assign rules to profiles without going into details.
+- Product-independent profiles (PIPs s.a. ANSSI, HIPAA, PCI-DSS, …) and respective control files - although PIP development that benefits the whole community instead of cluttering the content with “if product in [...]”.
+- Shared resources - rules only loosely coupled to products, templates, CPEs etc.
+- Build system - decisions upon architecture of the build system impact build time, project capabilities, and can also move maintenance costs.
+- Test-related code - ensure that tests have the greatest coverage as possible, but that don’t waste time and don’t suffer from false positives.
+- Organization administration - ability to grant and remove rights, create and delete repositories etc.
+
+
+## Implementation
+
+### Products
+
+- Keeping track of the decomposition: Metadata in product.yml files, CODEOWNERS file
```

Comment:
CODEOWNERS would be cool :-) Looks like we don't have any right now, so we'll probably have to ask for them retroactively in products? Might be cool to have Red Hat propose some for the existing content in parallel with this PR so we can see roughly what that'd look like both for RH properties and for CaC-the-build-system &c?

Reply:
Yes, we have to figure out how it works. You are right that initial version of CODEOWNERS should be a part of the PR when it is not a draft any more.